pages/blog/oss-contributor-workflow.mdx

```diff
@@ -0,0 +1,88 @@
+- [GraphQL Mesh](https://www.graphql-mesh.com/)
+- [GraphQL Tools](https://www.graphql-tools.com/)
+- [GraphQL Modules](https://graphql-modules.com/)
+- [Apollo Angular](https://apollo-angular.com/)
+- [GraphQL CLI](https://graphql-cli.com/introduction)
+- [GraphQL Config](https://graphql-config.com/introduction)
+- [SOFA](https://www.sofa-api.com/)
+- [GraphQL ESLint](https://github.com/dotansimha/graphql-eslint/)
+- [GraphQL Inspector](https://graphql-inspector.com/)
+- [GraphQL Hive](https://graphql-hive.com/)
+
+As you can see, the list goes on (for a good reason), and we have more interesting projects on the way which you can find [here](https://the-guild.dev/open-source) 😉
+
+And some of the major things which we were looking to do were to:
+
+- Provide a great experience for beginners who want to contribute to any of these projects
+- Give the maintainers an eagle eye view of all the projects and things to do to progress to the next steps
+- Provide a level of standardization in the contribution/development workflow of all these projects so that we don't have to re-invent the wheel every time, but rather spend time on the important stuff
+- Create a framework of sorts which works not only for the projects we work on in The Guild, but something which can work for other open source projects as well.
+
+This led us to the creation of the [contributor workflow guide](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md) to help us standardize the workflow, guide beginners who would like to contribute and also act as a guide for other open source projects who are looking at a similar standardization approach (What we have is an initial iteration of the guide and there is definitely a lot of room for improvement. If you have any suggestions, do let us know - we are all ears 🙂)
+
+But does creating a guide suffice to achieve our goal? As you guessed, definitely not, and the guide was just a start to an amazing journey ahead.
+
+The next thing we did was to prepare a set of [standard issue templates](https://github.com/Urigo/graphql-mesh/tree/master/.github/ISSUE_TEMPLATE) which remains almost the same across all our repositories. Again, a work in progress we will tweak this as the need arises. This gives us an ability to suggest a standard template to file issues and PRs to all the new contributors asking for all the important details like the environment and version they use, possible reproductions/screenshots, additional context and also providing them with checklists to be actioned upon when filing an issue.
+
+While all this might seem very simple, doing this has a long term benefit of communicating your expectations as a maintainer to the community when they are looking to contribute. Now, this is something which some of the projects tend to ignore.
+
+Then we started thinking about the next important thing which we can address to make both maintainer's and contributor's lives easier - LABELS
```

Comment:
Maybe talk about labels before issue templates?
It's a bigger change

Reply:
with header that's called something like "Process based Github Labels" or something?